app/classes/query/base.rb

```diff
@@ -65,6 +65,12 @@ def self.content_filter_parameters
 
   delegate :content_filter_parameters, to: :class
 
+  # def self.subquery_parameters
+  #   parameter_declarations.select { |key, _v| key.to_s.include?("_query") }
+  # end
+
+  # delegate :subquery_parameters, to: :class
+
```

Comment:
Keeping these accessors as comments because 
- I will need them on the faceted search PR to distinguish params that should go in the form from subquery params
- I keep adding and deleting these methods on various PRs, and each time i forget the syntax